charts/falcosidekick/values.yaml

```diff
@@ -722,6 +722,19 @@ config:
     checkcert: true
     # -- minimum priority of event to use this output, order is `emergency\|alert\|critical\|error\|warning\|notice\|informational\|debug or ""`
     minimumpriority: ""
+    # -- dashboards contains configuration for grafana dashboards.
+    dashboards:
+      falcosidekickLoki:
+        # -- enabled specifies whether this dashboard should be deployed.
+        enabled: false
+        # --configmaps to be deployed that contain a grafana dashboard.
+        configMap:
+          # -- name specifies the name for the configmap.
+          name: falcosidekick-loki-dashboard-grafana
+          # -- namespace specifies the namespace for the configmap.
+          namespace: ""
+          # -- folder where the dashboard is stored by grafana.
+          folder: ""
```

Comment:
there's a mismatch here, this output `grafana` creates annotations, this block should in the `Loki` section.

something like:

```yaml
  loki:
    # -- Loki <http://host:port>, if not `empty`, Loki is *enabled*
    hostport: ""
    # -- user for Grafana Logs
    user: ""
    # -- API Key for Grafana Logs
    apikey: ""
    # -- Loki endpoint URL path, more info: <https://grafana.com/docs/loki/latest/api/#post-apiprompush>
    endpoint: "/loki/api/v1/push"
    # -- Loki tenant, if not `empty`, Loki tenant is *enabled*
    tenant: ""
    # -- comma separated list of fields to use as labels additionally to rule, source, priority, tags and custom_fields
    extralabels: ""
    # -- a list of comma separated custom headers to add, syntax is "key:value,key:value"
    customheaders: ""
    # -- minimum priority of event to use this output, order is `emergency\|alert\|critical\|error\|warning\|notice\|informational\|debug or ""`
    minimumpriority: ""
    # -- if true, checkcert flag will be ignored (server cert will always be checked)
    mutualtls: false
    # -- check if ssl certificate of the output is valid
    checkcert: true
    # -- dashboard for Grafana
    grafanaDashboard:
      # -- enabled specifies whether this dashboard should be deployed.
      enabled: false
      # --configmaps to be deployed that contain a grafana dashboard.
      configMap:
        # -- name specifies the name for the configmap.
        name: falcosidekick-loki-dashboard-grafana
        # -- namespace specifies the namespace for the configmap.
        namespace: ""
        # -- folder where the dashboard is stored by grafana.
        folder: ""
```

Reply:
the change suggested is done, but one question. What do you mean that "grafana create annotations"??

Reply:
the section `grafana` is to configure the output of falcosidekick which creates annotations for the graphs with the falco events: https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/annotate-visualizations/